css/layout/product-footer.css

```diff
@@ -0,0 +1,10 @@
+.product-footer {
+    padding: 3rem;
+    background-color: var(--secondary-color);
+    color: var(--bg-color);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-weight: 400;
```

Comment:
Original font weight is 400 by default. In case if you are  not overriding css from component library, this can be omitted.